### ISSUE TITLE
eagle missing in word.txt

### ISSUE THREAD

**Issue by fshahinfar1** [dwyl/english-words#206]
word `eagle` is missing word.txt, although it is in word_alpha.txt . There is `Eagle` in word.txt which is a bit weird given that other words are not capitalized. If you want I can make a pull request fixing this.

**Comment by jmatos-dev** [comment]
Hello. Do you play Wordle?

**Comment by fshahinfar1** [comment]
Yes, you caught me! I narrowed the game to 6 (or so not really remembering) letters. Had one last word to guess and thought I could use grep on a text file :) . That is how I found this repo. Unfortunately I failed to cheat at wordle  :P

**Comment by jmatos-dev** [comment]
Hehehe. I also play Wordle. That morning, I got an email message with your comment and I didn't pay attention since, at the moment, to me, it was just a random issue with the repo. But when I solved Wordle that morning and the word was eagle, I immediately remembered your message and thought that it was not a coincidence.

So, how do you use the repo to help you with Wordle?

Do you play Spelling Bee? I wrote a script in Python than helps me with that game.
